enterprise/server/cmd/ci_runner/main.go

```diff
@@ -2471,13 +2471,15 @@ func configureGlobalCredentialHelper(ctx context.Context) error {
 // necessarily have any SSH private keys available that we can use to clone
 // repos over SSH.
 func configureGlobalURLRewrites(ctx context.Context) error {
+	flag := "--replace-all"
```

Comment:
```suggestion
		// Discard any existing rewrites initially, then append.
		flag := "--replace-all"
```

Reply:
hmm the indentation looks weird here on Github UI.
Let's see if it lands correctly

Reply:
It was a little off. Pulled + reformat + pushed